client/src/views/Profile.jsx

```diff
@@ -0,0 +1,351 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Container from "react-bootstrap/Container";
+import Profilephoto from "../assets/ImageLarge.png";
+import { useHistory } from "react-router-dom";
+import axios from "axios";
+import { FaCheckCircle, FaBullseye, FaAngleRight } from "react-icons/fa";
+const Profile = ({ auth }) => {
+  const history = useHistory();
```

Comment:
Use useContext here like what u did in Activity
```suggestion
  const history = useHistory();
  const { auth, setAuth } = useContext(AuthContext);
```